lib/vscode/src/vs/server/node/server.ts

```diff
@@ -212,11 +212,27 @@ export class Vscode {
 	}
 
 	private async initializeServices(args: NativeParsedArgs): Promise<void> {
+		/*
+			NOTE@coder: this initializeServices is loosely based off this file:
```

Comment:
Thanks for the example! 🙌 I don't mind making them relative. Will fix!

Re: comments
Ah, right this is our file. I only added them because I know future me would want to remember why/what we changed so that's why I added them.

But happy to remove them if you think we should omit!